### PR TITLE
Update READMEs for Compute

### PR DESCRIPTION
**Pull request by andreamlin**
- Update READMEs and URL links.
- Add `ComputeExample.java` which contains working client code for the Compute client.

Resolves https://github.com/GoogleCloudPlatform/google-cloud-java/issues/3377

### CODE REVIEW

**Comment by andreamlin** [comment]
PTAL

File: google-cloud-clients/google-cloud-compute/README.md [hunk 1]
```diff
@@ -12,7 +12,6 @@ Java idiomatic client for [Google Cloud Compute][cloud-compute].
 - [Product Documentation][compute-product-docs]
 - [Client Library Documentation][compute-client-lib-docs]
 
-> Note: This client is no longer receiving updates; new features in the Compute API will not be added to this client.
```
Comment (garrettjonesgoogle) [inline google-cloud-clients/google-cloud-compute/README.md]: The code samples in this file still use the deprecated code.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/compute/v1/ComputeExample.java [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+import com.google.cloud.compute.v1.ListAddressesHttpRequest;
+import com.google.cloud.compute.v1.Operation;
+import com.google.cloud.compute.v1.ProjectRegionAddressName;
+import com.google.cloud.compute.v1.ProjectRegionName;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+
+/** Working example code to make live calls on Addresses resources in a GCP Compute project. */
+public class ComputeExample {
```
Comment (garrettjonesgoogle) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/compute/v1/ComputeExample.java]: Can you find the DPE for Compute to look over this?

File: google-cloud-clients/google-cloud-compute/README.md (hunk 1 above)
Comment (andreamlin) [reply google-cloud-clients/google-cloud-compute/README.md]: Blocked by https://github.com/googleapis/gapic-generator/issues/2096

Comment (JustinBeckwith) [reply google-cloud-clients/google-cloud-compute/README.md]: Looks like this was resolved 😄

File: google-cloud-examples/src/main/java/com/google/cloud/examples/compute/v1/ComputeExample.java (hunk 2 above)
Comment (JustinBeckwith) [reply google-cloud-examples/src/main/java/com/google/cloud/examples/compute/v1/ComputeExample.java]: @djmailhot - would you mind giving this a look over?

**Comment by JustinBeckwith** [comment]
It would be rad to merge this in, and then submit a follow-up PR if we get feedback :)

**Pull request merged**